__tests__/app.test.js

```diff
@@ -114,7 +122,60 @@ describe('GET: /api/articles', () => {
         .get('/api/articles')
         .expect(200)
         .then(({body}) => {
-            expect(body).toBeSortedBy("created_at", {descending: true}, {coerce: true})
+            body.forEach((article) => {
+                const regex = /[^0-9]/g
+                const stringCreated = article.created_at.replace(regex, "")
+                article.created_at = stringCreated
+            })
+            expect(body).toBeSortedBy('created_at', {descending: true})
+        })
+    })
+})
+describe('GET: /api/articles/:article_id/comments',() => {
+    test('GET 200: Returns an array of comment objects with the correct properties', () => {
+        return request(app)
+        .get('/api/articles/5/comments')
+        .expect(200)
+        .then(({body}) => {
+            body.forEach((comment) => {
```

Comment:
Don't forget to add a check for the length of the array - so that you don't get a falsely passing test if the returned array happens to be empty 🔍